dev/bin/knative/resources/knative-eventing.yaml

```diff
@@ -3275,12 +3279,12 @@ metadata:
   labels:
     eventing.knative.dev/release: devel
   annotations:
-    knative.dev/example-checksum: "f8e5a744"
+    knative.dev/example-checksum: "9185c153"
```

Comment:
Is this hash mandatory? It seems a little bit suspicious to me to add some hashes there. Can you please explain.

Reply:
@jiripetrlik this file is copied from here https://github.com/openshift-knative/eventing-kafka-broker/blob/release-v1.2/openshift/release/artifacts/eventing-kafka.yaml . It's just the standard knative installation. 

Was there a specific reason not to use directly the link https://github.com/openshift-knative/eventing-kafka-broker/blob/release-v1.2/openshift/release/artifacts/eventing-kafka.yaml in the installation script? At the end the link points to a release, I think it should be fine just to keep this reference to a fixed version. wdyt @matzew ?